jupyter_server/gateway/managers.py

```diff
@@ -632,9 +632,10 @@ async def get_msg(self, *args: Any, **kwargs: Any) -> dict[str, Any]:
         timeout = kwargs.get("timeout", 1)
         msg = await self._async_get(timeout=timeout)
         self.log.debug(
-            "Received message on channel: {}, msg_id: {}, msg_type: {}".format(
-                self.channel_name, msg["msg_id"], msg["msg_type"] if msg else "null"
-            )
+            "Received message on channel: {}, msg_id: {}, msg_type: {}",
+            self.channel_name,
+            msg["msg_id"],
+            msg["msg_type"] if msg else "null",
```

Comment:
`jupyter_server/gateway/managers.py:634:9: PLE1205 Too many arguments for `logging` format string`

```suggestion
            "Received message on channel: %s, msg_id: %s, msg_type: %s",
            self.channel_name,
            msg["msg_id"],
            msg["msg_type"] if msg else "null",
```

Reply:
thanks !